### PR TITLE
Edit codeowner permissions

### PR DESCRIPTION
**Pull request by iwysiu**
Reduces our (the grafana team's) code ownership to just the github files, the go code, and the frontend datasource code. Hopefully this improves the Twinmaker team's development experience.

Notes:
According to [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) I'm fairly sure the un-owned code will not have us as code owners and thus will not require our approval to be merged. I can add other files if we think that we should be controlling access to them.

### CODE REVIEW

**Comment by idastambuk** [comment]
It's cool we're able to do this! Does this mean theoretically any community member will be able to merge without approval? I guess it's not a huge issue since only we can release the plugin, but doesn't this mean things might slip through the cracks more easily and [potentially](https://docs.google.com/document/d/1HGQXt-CLX4ME_CcW2u6Fmkuyj-mXq96RLoJf51xz-vs/edit#) cause a security issue? 
Would it be possible to add the Twinmaker team as code owners of other files? Though this would probably be more maintenance work in the long run

**Comment by iwysiu** [comment]
My understanding is that you need to be an approved contributor to merge, and we have main set to require approvals. We have the Twinmaker team members as outside contributors to the repo, which is why they have permissions. We could add them as code owners of other files but that would probably be wordy, since they aren't in a grouping of any sort (and I don't think we can group them)

**Comment by idastambuk** [comment]
> My understanding is that you need to be an approved contributor to merge, and we have main set to require approvals. We have the Twinmaker team members as outside contributors to the repo, which is why they have permissions. We could add them as code owners of other files but that would probably be wordy, since they aren't in a grouping of any sort (and I don't think we can group them)

Didn't know this! Makes sense, I think this PR is then enough 🚀

**Review by idastambuk — APPROVED** [review]

**Pull request merged**
